usvm-go/src/main/kotlin/org/usvm/machine/GoMachine.kt

```diff
@@ -0,0 +1,129 @@
+package org.usvm.machine
+
+import mu.KLogging
+import org.usvm.StateCollectionStrategy
+import org.usvm.UMachine
+import org.usvm.UMachineOptions
+import org.usvm.bridge.GoBridge
+import org.usvm.machine.interpreter.GoInterpreter
+import org.usvm.machine.state.GoMethodResult
+import org.usvm.machine.state.GoState
+import org.usvm.ps.createPathSelector
+import org.usvm.statistics.*
```

Comment:
We don't use asterisk in our code style, could you change it for kt files please?